install_dependencies.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
 
 if [[ -z $HYRISE_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost-dev 2>/dev/null | grep "install ok installed")
```

Comment:
Could you please add a comment what this is doing so I do not have to go to the trird nested if to find out what's happening?

Reply:
Removed. Removing is apparently no longer necessary.